packages/core/components/Icons/LoadingIcon.tsx

```diff
@@ -0,0 +1,25 @@
+import { Spinner, SpinnerSize } from "@fluentui/react";
+import classNames from "classnames";
+import * as React from "react";
+
+import styles from "./LoadingIcon.module.css";
+
+interface Props {
+    className?: string;
+    invertColor?: boolean;
+    size?: SpinnerSize;
+
+    "data-testid"?: string;
```

Comment:
what is this for?

Reply:
This is a prop used by the testing components - generally we don't need to explicitly define it like this but in this case we do